### PR TITLE
Add a poster feature to match with modelviewer.dev/example/tester

### PR DESCRIPTION
**Pull request by ziyanma**
Add a create / display poster feature.

Tested workflow: 

1. Upload a model.
2. Click `create poster` in pencil tab, user will see a poster in the output snippet.
3. Click `display poster` in pencil tab, user will see the poster previewed in model-viewer-preview
4. Upload another model, the previous poster settings should be cleared.
5. Upload a snippet with poster (with reveal="interaction" potentially), user should see the poster before model loaded (or interacted with poster)

### CODE REVIEW

File: packages/space-opera/src/components/poster_controls/poster_controls.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import '../shared/expandable_content/expandable_tab.js';
```
Comment (stevesan) [inline packages/space-opera/src/components/poster_controls/poster_controls.ts:1]: Add the copyright notice at the top

File: packages/space-opera/src/redux/poster_dispatcher.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {registerStateMutator, State} from './space_opera_base';
```
Comment (stevesan) [inline packages/space-opera/src/redux/poster_dispatcher.ts:1]: rename this file to poster_dispatchers - plural.

File: packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts [hunk 3]
```diff
@@ -61,21 +61,23 @@ export class ExportPanel extends ConnectedLitElement {
...
     }
 
     if (editedConfig.environmentImage &&
-      isObjectUrl(editedConfig.environmentImage)) {
+        isObjectUrl(editedConfig.environmentImage)) {
       // Uploaded env image
       editedConfig.environmentImage = `Change this to your HDR URL`;
     }
-
+    if (editedConfig.poster) {
```
[the remark this answers is not in the document]
Comment (ziyanma) [reply packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts:78]: Well it does, but we should also handle the env-image / glb the same way. May do that in a different PR

[the remark this answers is not in the document]
Comment (stevesan) [reply packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts:78]: didn't i already do that? how do you mean?

File: packages/space-opera/src/redux/poster_dispatcher.ts (hunk 2 above)
Comment (ziyanma) [reply packages/space-opera/src/redux/poster_dispatcher.ts]: Done

File: packages/space-opera/src/components/poster_controls/poster_controls.ts (hunk 1 above)
Comment (ziyanma) [reply packages/space-opera/src/components/poster_controls/poster_controls.ts:19]: Weird I only did this to the .css.ts :)

File: packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ziyanma) [reply packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts]: Ohh. they already changed without my notice. Updated

**Comment by stevesan** [comment]
James, could you create a non-master branch for this? We need that to run full tests.

**Pull request merged**